linter/linter.go

```diff
@@ -581,6 +605,32 @@ func (l *Linter) lintSubRoutineDeclaration(decl *ast.SubroutineDeclaration, ctx
 	return types.NeverType
 }
 
+func (l *Linter) lintPenaltyboxDeclaration(decl *ast.PenaltyboxDeclaration) types.Type {
+	// validate penaltybox name
+	if !isValidName(decl.Name.Value) {
+		l.Error(InvalidName(decl.Name.GetMeta(), decl.Name.Value, "penaltybox").Match(PENALTYBOX_SYNTAX))
+	}
+
+	if len(decl.Block.Statements) > 0 {
+		l.Error(NonEmptyPenaltyboxBlock(decl.GetMeta(), decl.Name.Value))
```

Comment:
Please link the rule matching that you added to `PENALTYBOX_SYNTAX`.
(Or an Individual rule like `PENALTYBOX_MUST_BE_EMPTY` is fine, maybe)

```suggestion
		l.Error(NonEmptyPenaltyboxBlock(decl.GetMeta(), decl.Name.Value).Match(PENALTYBOX_SYNTAX))
```